src/main/java/uk/gov/companieshouse/pscstatement/delta/mapper/StatementMapper.java

```diff
@@ -29,7 +39,8 @@ public interface StatementMapper {
     @Mapping(target = "links", source = "companyNumber", ignore = true)
     @Mapping(target = "linkedPscName", source = "linkedPsc.surname", ignore = true)
     @Mapping(target = "notifiedOn", source = "submittedOn", dateFormat = "yyyyMMdd")
-    @Mapping(target = "restrictionsNoticeWithdrawalReason", source = "restrictionsNoticeReason")
+    @Mapping(target = "restrictionsNoticeWithdrawalReason", source = "restrictionsNoticeReason",
```

Comment:
Could we just ignore the mapping if using an after mapping for the restrictions anyway?